### PR TITLE
Add staticwebapp.config.json schema

### PR DESCRIPTION
**Pull request by wwlorey**
Related to https://github.com/microsoft/vscode-azurestaticwebapps/issues/97
Fixes https://github.com/microsoft/vscode-azurestaticwebapps/issues/98

### CODE REVIEW

**Review by nturinski — APPROVED** [review]
This is really awesome/helpful 👍

File: package.json [hunk 1]
```diff
@@ -383,6 +383,12 @@
                 }
             ]
         },
+        "jsonValidation": [
+            {
+                "fileMatch": "staticwebapp.config.json",
```
Comment (nturinski) [inline package.json:388]: Change this to 
`/staticwebapp.config.json`

Eric has it this way in Functions, I think it's so it doesn't _have_ to be in the root.

Comment (nturinski) [inline package.json:388]: Eric has it like `/function.json` in the Functions repo, but I'm not really sure what the purpose/reason is.  @ejizba Is there any reason that we should fileMatch with a slash prefix?

Comment (ejizba) [reply package.json:388]: I don't remember. Only thing I can think of is maybe the slash prevents something like "test.function.json" from being affected - but I tested and doesn't seem to make a difference.

**Comment by wwlorey** [comment]
I just tested on my end as well - the file was recognized in a subdirectory or in the root dir

**Pull request merged**